### ISSUE TITLE
Hello

### ISSUE THREAD

**Issue by Nakilon** [cguess/eikon#7]
May I wonder what didn't work for you in dhash-vips?
Is it because https://github.com/Nakilon/dhash-vips/issues/12 kept erroring on ARM?

**Comment by Nakilon** [comment]
And a note: since you've also switched from idhash to dhash the quality of comparison can degrade a little.

**Comment by cguess** [comment]
@Nakilon correct, at least at the time ARM was breaking so it made it impossible to test during development on my MacBook Pro M1's. I haven't gone back and looked further however since this is working quite well for the moment.

Thank you for the tip on the algorithm, I may take some time in the next few weeks to update.

If you'd like me to help with anything just let me know.

**Comment by Nakilon** [comment]
IIRC it was weird that it has compiled the extension but it didn't see the methods it had to have.
Not sure what was going on there but probably there is some simple ruby way to add some `--without-extension` option passable to gem install or Gemfile so you could just force it to skip trying to build the extensions and fallback to ruby implementation. It was meant to be able to doctor itself for the compilation errors but the error is weird.

Is it still failing? Probably ruby and macos had some updates.
